src/Containers/App.jsx

```diff
@@ -12,7 +19,8 @@ const App = () => {
             <Route exact path='/' component={CardList} />
             <Route exact path='/sing-in' component={SingIn} />
             <Route exact path='/card/:id' component={FullCard} />
-            <Route path='/404' component={PageNotFound} />
+            <Route exact path='/settings' component={Settings} />
```

Comment:
Basically there is no sense in rendering such a route for non-admin user since it routes to redirect. Simply do not render it for such case.